liquibase-standard/src/main/java/liquibase/command/core/DiffChangelogCommandStep.java

```diff
@@ -43,6 +46,11 @@ public class DiffChangelogCommandStep extends AbstractCommandStep {
                 .addAlias("contexts")
                 .description("Changeset contexts to generate")
                 .build();
+        USE_OR_REPLACE_OPTION = builder.argument("useOrReplaceOption", Boolean.class)
+                .description("If true, will add 'OR REPLACE' option to the create view change object")
+                .defaultValue(false)
+                .setValueHandler(ValueHandlerUtil::booleanValueHandler)
+                .build();
```

Comment:
It isn't necessary for this PR, but instead of repeating the argument, you can make a new class that extends `AbstractArgumentCommandStep`, and then specify that as a `requiresDependency` here and in every other class which requires this argument. This helps to reduce the code duplication.

Reply:
I created an abstract class with some common behavior that `GenerateChangelogCommandStep` and `DiffChangelogCommandStep` both use, I was thinking once that [PR](https://github.com/liquibase/liquibase/pull/4635/files) is merged to move this new argument there, and then in these two command classes do this:
```
builder.addArgument(AbstractChangelogCommandStep. USE_OR_REPLACE_OPTION).build();
```
What do you think about it?

Thanks,
Daniel.

Reply:
That sounds good. I don't think that's been done before, but it sounds simpler.

Reply:
It is! @filipelautert showed me yesterday that was already applied to `RollbackCommandStep`, so I applied the same to GenerateChangelog and DiffChangelog command step classes.